src/elements/content-sharing/hooks/useCollaborators.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import * as React from 'react';
+import API from '../../../api';
+import { convertCollabsResponse } from '../../../features/unified-share-modal/utils/convertData';
+import { TYPE_FILE, TYPE_FOLDER } from '../../../constants';
+import type { Collaborations, ItemType } from '../../../common/types/core';
+import type { collaboratorsListType } from '../../../features/unified-share-modal/flowTypes';
+
+/**
+ * Get the item's collaborators
+ *
+ * A note on the wording: the USM uses the term "collaborators" internally,
+ * so the state variable and state setting function also refer to "collaborators."
+ * However, we are using the Collaborations API here, so the API-related functions
+ * use the term "collaborations." For more details, see ./api/FileCollaborations.
+ *
+ * @param {API} api
+ * @param {string} itemID
+ * @param {ItemType} itemType
+ * @param {string} ownerEmail
+ * @param {boolean} isCurrentUserOwner
+ * @param {Function} [handleSuccess]
+ * @param {Function} [handleError]
+ * @returns {collaboratorsListType | null}
+ */
+function useCollaborators(
```

Comment:
now that these hooks are separated out, do we need unit tests for them or are they covered by the tests in ContentSharing.test.js?

Reply:
They are covered by ContentSharing.test.js and SharingNotification.test.js, but I'll add more tests specifically for the hooks.